webapp/src/components/call_widget/loading_overlay.tsx

```diff
@@ -1,77 +0,0 @@
-// Copyright (c) 2020-present Mattermost, Inc. All Rights Reserved.
-// See LICENSE.txt for license information.
-
-import React, {useMemo, useState} from 'react';
-import {useIntl} from 'react-intl';
-import {Spinner} from 'src/components/shared';
-import styled, {css} from 'styled-components';
-
-export type Props = {
-    visible: boolean,
-    joining: boolean,
-}
-
-export default function LoadingOverlay({visible, joining}: Props) {
-    const {formatMessage} = useIntl();
-    const [animationEnded, setAnimationEnded] = useState(false);
-    const wasJoining = useMemo(() => {
-        return joining;
-    }, [/* intentionally empty */]);
-
-    if (!visible && animationEnded) {
-        return null;
-    }
-
-    const onAnimationEnd = () => {
-        setAnimationEnded(true);
-    };
```

Comment:
This would never fire and leave the component mounted for the whole duration of the call at zero opacity. I am fixing it as part of this PR.